addons/mail/migrations/13.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,115 @@
+
+mail         / ir.model                 / is_mail_activity (boolean)    : NEW hasdefault
+mail         / ir.model                 / is_mail_blacklist (boolean)   : NEW hasdefault
+# Nothing to do
+
+mail         / ir.model.fields          / track_visibility (selection)  : DEL selection_keys: ['always', 'onchange']
+mail         / ir.model.fields          / tracking (integer)            : NEW
+# Done in post-migration script
+
+mail         / mail.activity            / create_user_id (many2one)     : DEL relation: res.users
+mail         / mail.activity            / feedback (html)               : DEL
+mail         / mail.activity            / res_id (integer)              : relation is now 'res_model' ('False') [nothing to do]
+mail         / mail.activity            / res_id (integer)              : type is now 'many2one_reference' ('integer')
+# Nothing to do
+
+mail         / mail.activity.type       / category (selection)          : selection_keys is now '['default', 'upload_file']' ('['default']')
+# Nothing to do
+
+mail         / mail.activity.type       / default_description (html)    : NEW
+mail         / mail.activity.type       / default_user_id (many2one)    : NEW relation: res.users
+# Nothing to do. Use as default values for note and user_id fields of mail.activity model
+
+mail         / mail.channel             / image (binary)                : DEL attachment: True [renamed to image_128]
+mail         / mail.channel             / image_128 (binary)            : NEW attachment: True, hasdefault [renamed from image]
+# Done in pre-migration script
+
+mail         / mail.channel             / image_medium (binary)         : DEL attachment: True
+mail         / mail.channel             / image_small (binary)          : DEL attachment: True
+# Nothing to do
+
+mail         / mail.channel.partner     / custom_channel_name (char)    : NEW
+# Nothing to do
+
+mail         / mail.channel.partner     / fetched_message_id (many2one) : NEW relation: mail.message
+# I think nothing to do
+# Commit https://github.com/odoo/odoo/commit/844fb2b51ea7cfa557b19466c0a53dde19433e7a
+
+mail         / mail.followers           / res_id (integer)              : relation is now 'res_model' ('False') [nothing to do]
+mail         / mail.followers           / res_id (integer)              : type is now 'many2one_reference' ('integer')
+# Nothing to do
+
+mail         / mail.mail                / res_id (integer)              : relation is now 'model' ('False') [nothing to do]
+mail         / mail.mail                / res_id (integer)              : type is now 'many2one_reference' ('integer')
+# Nothing to do
+
+mail         / mail.message             / mail_ids (one2many)           : NEW relation: mail.mail
+# Nothing to do
+
+mail         / mail.message             / message_type (selection)      : selection_keys is now '['comment', 'email', 'notification', 'user_notification']' ('['comment', 'email', 'notification']')
+# I think nothing to do
```

Comment:
I think you can set message_type to `user_notification` where message_type = `notification` and message_id like '%openerp-message-notify%';